src/vasc/vasculature.cpp

```diff
@@ -18,7 +18,9 @@ Vasculature::Vasculature(const std::string& source) {
         throw UnknownFileType("File has no extension");
     }
 
-    if (access(source.c_str(), F_OK) == -1) {
+    // Cross-platform check of file existance
+    std::ifstream file(source.c_str());
+    if (!file) {
```

Comment:
file might exist, you but you don't have permissions to read it, at which point the error msg is misleading